src/webcomponents/commons/filters/ontology-autocomplete-filter.js

```diff
@@ -0,0 +1,120 @@
+/**
+ * Copyright 2015-2019 OpenCB
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {LitElement, html} from "lit";
+import "../../commons/forms/select-token-filter.js";
+import LitUtils from "../utils/lit-utils.js";
+import NotificationUtils from "../utils/notification-utils.js";
+import BioinfoUtils from "../../../core/bioinfo/bioinfo-utils.js";
+
+export default class OntologyAutocompleteFilter extends LitElement {
+
+    constructor() {
+        super();
+        this.#init();
+    }
+
+    createRenderRoot() {
+        return this;
+    }
+
+    static get properties() {
+        return {
+            source: {
+                type: String
+            },
+            value: {
+                type: Object
+            },
+            config: {
+                type: Object
+            }
+        };
+    }
+
+    #init() {
+        this.ontologyConfig = {
+            root: "https://ws.zettagenomics.com/cellbase/webservices/rest/v5/hsapiens/feature/ontology",
+            search: "/search",
+        };
```

Comment:
The right place for this configuration should be in `getDefaultConfig`.

Also, as I see this is only used in line 85 so maybe you do not need to declare it as a variable of the class.